emcc

```diff
@@ -1275,6 +1288,9 @@ try:
 
   # Emscripten
   logging.debug('LLVM => JS')
+  for lib in library_files:
```

Comment:
OK, I've done some perf testing for you to check this on a couple of sub-projects of our various products which use emscripten.

I've used three build options:

1. Stock emscripten (incoming). All jso files excluded from the build/link because it doesn't understand them
1. Modified emscripten (this pull request). All jso files excluded from the build/link for comparison purposes
1. Modified emscripten, compiling and linking in the our jso files as well

All of these are using plain make, "-j 1", clean builds, timed after running CMake to set up the makefiles. The machine has an SSD.

### Project 1 - Debug build, 338 C++ files, 375 headers, uses a couple of static libraries

option | times | average
-------|-------|--------
stock emscripten, jso excluded | 1:09.0, 1:08.8, 1:09:2 | 69.0
new emscripten, jso excluded | 1:09.6, 1:09.2, 1:08.8 | 69.2
new emscripten, jso included | 1:10.7, 1:10.2, 1:10.0 | 70.3

### Project 2 - RelWithDebInfo build, ~600 C++ files, uses quite a few static libraries

option | times | average
-------|-------|--------
stock emscripten, jso excluded | 2:37.6, 2:37.0, 2:37.5 | 157.4
new emscripten, jso excluded | 2:37.7, 2:37.1, 2:37.4 | 157.4
new emscripten, jso included | 2:38.8, 2:38.8, 2:37.9 | 158.5

### Conclusion

Adding jso files significantly increases link time! That's actually a bit more of an effect than I'd anticipated. But, there's no significant increase in time with/without these changes for projects that aren't using JS libraries.